dags/homelessness/sync_rap_data.py

```diff
@@ -0,0 +1,95 @@
+"""
+Pull data from RAP FTP and sync with ArcGIS Online
+"""
+import pandas as pd
+import ftplib
+import arcgis
```

Comment:
@brendanbailey Can you add this to the `requirements.txt` file?

Reply:
Adding Arcgis. Pandas is already in there, and ftplib is a built in library